docs/content/preview/deploy/_index.md

```diff
@@ -2,8 +2,8 @@
 title: Deploy
 headerTitle: Deploy YugabyteDB
 linkTitle: Deploy
-description: Deploy to any public cloud or private data center of your choice.
-headcontent: Deploy to the public cloud or private data center of your choice
+description: How to deploy the YugabyteDB database to any public cloud or private data center or Kubernetes. Includes checklist and manual deployment options as well.
+headcontent: How to deploy the YugabyteDB database to any public cloud or private data center or Kubernetes. Includes checklist and manual deployment options as well.
```

Comment:
SEO changes concern title and description fields only. Generally, update these only.
headcontent is the subtitle that is displayed on the page, and can't be long like that description; they don't have to be the same though they often are if not too long. Here, a good headcontent would be "Deploy to the public cloud, a private data center, or Kubernetes" No periods on headcontent.